src/components/MegaButton/__tests__/MegaButton.spec.js

```diff
@@ -161,7 +195,107 @@ describe('Megabutton', () => {
       });
 
       const text = getByText(props.text);
-      expect(text).toBeValid();
+      expect(text).not.toBeInTheDocument;
+    });
+
+  });
+
+  describe('Small text with image', () => {
+
+    const textWithImageProps = {
+      id: 'megabutton',
+      modelValue: '',
+      name: 'Fine print',
+      label: 'Megabutton with text',
+      text: 'This also has an image',
+      imageSource: 'fake_image.egg',
+      value: 'bigOlButton'
+    };
+
+    it('renders the label in smaller text', () => {
+      const props = textWithImageProps;
+
+      const { getByText } = render(MegaButton, {
+        props
+      });
+
+      const label = getByText(props.label);
+      expect(label).toHaveClass('text-lg');
+    });
+
+    it('renders the text', () => {
+      const props = textWithImageProps;
+
+      const { getByText } = render(MegaButton, {
+        props
+      });
+
+      const text = getByText(props.text);
+      expect(text).not.toBeInTheDocument;
+    });
+
+    it('displays the image', () => {
+      const props = textWithImageProps;
+
+      const { getByTestId } = render(MegaButton, {
+        props
+      });
+
+      const imageContainer = getByTestId('imageContainer');
+      expect(imageContainer).not.toBeInTheDocument;
```

Comment:
Ok these have me confused now, if we want to test that "it renders the text" and "it displays the image" then I would expect they would both be in the document, whereas this is testing that they are not in the document.

Reply:
I am very confused as well, since they seem to pass regardless of whether it's `toBeInTheDocument` or `.toThrow`, or whether either has `.not` in front of it

Reply:
...wait. it's `.toBeInTheDocument()`

why it would pass regardless without the parens is beyond me